picknmix/picknmix.py

```diff
@@ -9,6 +9,7 @@
 from copy import deepcopy
 import numpy as np
 import warnings
+import sklearn
```

Comment:
All look great, however, to make it better, we can first check if the layer is composed of any Scikit-learn model first. You can refer to how `sklearn.base.clone` do it in their [source code](https://github.com/scikit-learn/scikit-learn/blob/1495f6924/sklearn/base.py#L32) basically they are checking if there is a `get_params` method. After that, we can import `sklearn.base.clone` by `__import__` method. I think this will also eliminate some dependency issue.

Of cause, in this case, for a non-scikit-learn object we have to make a deepcopy instead.

Reply:
I will do it. using importlib (equal to __import__) sounds like a very good idea. Gimme some time and I will implement it too. Another ticket wont be neccessary

Reply:
@Cheukting Done and done. Dependency is gone! :)